SPECS/kexec-tools/kexec-tools.spec

```diff
@@ -215,6 +218,11 @@ cp %{SOURCE109} $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlyk
 chmod 755 $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlykdump/%{remove_dracut_prefix %{SOURCE108}}
 chmod 755 $RPM_BUILD_ROOT/etc/kdump-adv-conf/kdump_dracut_modules/99earlykdump/%{remove_dracut_early_kdump_prefix %{SOURCE109}}
 
+# Add kexec-tools-specific boot configurations to /etc/default/grub.d
+# This configuration sets the crashkernel space allocated at boot
+# to the AzureLinux default value
+mkdir -p %{buildroot}%{_sysconfdir}/default/grub.d
+install -m 750 %{SOURCE30} %{buildroot}%{_sysconfdir}/default/grub.d/51_kexec_tools.cfg
```

Comment:
you don't need to mkdir, just use -D with install. Also, please don't set mode 750, it should be 755.